streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -466,7 +469,10 @@ public Set<String> makeReady(final Map<String, InternalTopicConfig> topics) {
                                                 topicName)
                                         );
                                     }
-                                } else {
+                                } else if (cause instanceof  TimeoutException) {
```

Comment:
I actually believe, we can remove the `catch TimeoutException` block below, because `get()` should never throw a timeout.

Reply:
ah -- that's right. done.